.github/workflows/main.yml

```diff
@@ -16,6 +16,16 @@ jobs:
         python-version: [ 3.6, 3.7 ]
         torch-version: [ 1.5.0, 1.6.0 ]
         tensorflow-version: [ 1.15.0 ]
+        include:
```

Comment:
Can we add `3.8`? Missing 3.8 between these versions is a bit weird.

Reply:
Looks like Python 3.8 requires TF 2.2: https://www.tensorflow.org/install/pip#system-requirements